docs/quickstart/setup.rst

```diff
@@ -50,6 +54,63 @@ networks can overlap, and nodes need not have distinct IPs for each plane.
 In order to reach the cluster-provided UIs from your host, the host needs to be
 able to connect to workload-plane IPs of the machines.
 
+Repositories provisioning
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Each machine needs to have repositories properly configured and access to basic
+repositories packages (depending on the operating systems).
+
+CentOS_:
+
+    - base
+    - extras
+    - updates
+
+RHEL_:
+
+    - rhel-7-server-rpms
+    - rhel-7-server-extras-rpms
+    - rhel-7-server-optional-rpms
+
+    .. note::
+
+        For RHEL_ you should have `a system properly registered`_.
+
+.. note::
+
+    The repositories names and configuration not necessarily need to be the
+    same as the official ones but all packages need to be available.
```

Comment:
The repository names and configurations do not necessarily need to be the same as the official ones but all packages need to be made available.